test/e2e/pkg/workspaces/workspaces.go

```diff
@@ -0,0 +1,42 @@
+package workspaces
+
+import (
+	"fmt"
+
+	"github.com/che-incubator/che-workspace-operator/test/e2e/pkg/metadata"
+)
+
+func (w *CodeReady) CreatePluginRegistryDeployment() (err error) {
+	label := "app=che-plugin-registry"
+	deployment, err := deserializePluginRegistryDeployment()
+
+	if err != nil {
+		fmt.Println("Failed to deserialize deployment")
+		return err
+	}
+
+	deployment, err = w.Kube().AppsV1().Deployments(metadata.Namespace.Name).Create(deployment)
+
+	if err != nil {
+		fmt.Println("Failed to create deployment %s: %s", deployment.Name, err)
+		return err
+	}
+
+	deploy, err := w.PodDeployWaitUtil(label)
+	if !deploy {
+		fmt.Println("Che Workspaces Controller not deployed")
+		return err
+	}
+	return nil
+}
+
+func (w *CodeReady) CreatePluginRegistryService() (err error) {
```

Comment:
I don't know how useful a named return is here, it'd be cleaner to not have it. It's also cleaner to not define vars here unless necessary, as it makes it easier to miss an error check.